src/ui/ConfigPanel.module.css

```
@@ -63,41 +63,96 @@
+}
+.sectionHeader:hover { color: var(--wc-text-muted); background: var(--wc-surface-2); }
+
+.chevron {
   flex-shrink: 0;
-  overflow-x: auto;
-  overflow-y: hidden;
-  scrollbar-width: thin;
-  scroll-behavior: smooth;
-  overscroll-behavior-x: contain;
+  transition: transform 0.15s;
+  color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)